cmd/spiffe-helper/config/config.go

```diff
@@ -158,9 +163,31 @@ func (c *Config) ValidateConfig(log logrus.FieldLogger) error {
 		c.JWTSVIDFileMode = defaultJWTSVIDFileMode
 	}
 
+	if c.HealthCheck.ListenerEnabled {
+		if c.HealthCheck.BindPort < 0 {
+			return errors.New("bind port must be positive")
+		}
+		if c.HealthCheck.BindPort == 0 {
+			c.HealthCheck.BindPort = defaultBindPort
+		}
+		if c.HealthCheck.HealthPath == "" {
+			c.HealthCheck.HealthPath = defaultHealthPath
+		}
+	}
+
 	return nil
 }
 
+func ParseConfigFile(log logrus.FieldLogger, configFile string, daemonModeFlag bool) (*Config, error) {
```

Comment:
The original function parses the actual config file so that should be named `ParseConfigFile()`. This new one can be name `ParseConfig()`